sycl/test-e2e/Assert/assert_in_kernels.cpp

```diff
@@ -2,6 +2,9 @@
 
 // https://github.com/intel/llvm/issues/7634
 // UNSUPPORTED: hip
+// TODO: Remove unsupported after fixing
```

Comment:
I'm not sure, but it looks like we shouldn't turn off Assert/* tests on acc as it's some environment issue, see https://github.com/intel/llvm/issues/12683#issuecomment-1941130187

Reply:
Could you try to change
`// RUN: %{run} %t.out &> %t.txt ; FileCheck %s --input-file %t.txt %if acc %{ --check-prefix=CHECK-ACC %}`
to
`// RUN: %{run} %t.out &> %t.txt ; FileCheck %s --input-file %t.txt %if accelerator %{ --check-prefix=CHECK-ACC %}`
instead of adding `UNSUPPORTED` to Assert/* tests?

Reply:
it should be %fpga